[feature] Create route for players

Adritay opened the pull request:
# Description

Added a player-route file.
Added a player-model file.
Edited the index.js file.

Adritay committed:
[feature] Create route for players
Description
========
Added a player-route file.
Added a player-model file.
Edited the index.js file.

david-suarez commented on models/player-model.js:
```diff
@@ -0,0 +1,43 @@
+        'default': "",
+        'required': true
+    },
+    'admin': Boolean
```
It is necessary this field in the Player Model?

david-suarez commented on models/player-model.js:
```diff
@@ -0,0 +1,43 @@
+});
+
+var PlayerModel = mongoose.model('Player', PlayerSchema);
+module.exports = PlayerModel;
```
Please add a new line add the end of the each file.

david-suarez commented on routes/api-v1.0/player-route.js:
```diff
@@ -0,0 +1,76 @@
+        query.exec(function(error, data) {
+            //console.log(data[0].name);
+            //console.log(data[0].lastname);
+            //console.log(data[0].address);
```
Remove useless code

david-suarez commented on routes/api-v1.0/player-route.js:
```diff
@@ -0,0 +1,76 @@
+        //var deferred = Q.defer();
+        newPlayer = request.body;
+        //console.log(request.body);
+        //if(newUser !== undefined) {
```
Remove useless code.

david-suarez commented on routes/api-v1.0/player-route.js:
```diff
@@ -0,0 +1,76 @@
+        });
+        //} else {
+        //    response.json('400', {'message': 'Bad Request'});
+        //}
```
Remove useless code.

david-suarez commented on routes/api-v1.0/player-route.js:
```diff
@@ -0,0 +1,76 @@
+    app.post(route.Players_Route, playerRoute.savePlayer);
+    //app.post(route.UserUploadImageRoute, userRoute.uploadUserImage);
+    //app["delete"](route.UserRoute, userRoute.removeUser);
+};
```
Add new line at the end of each file

david-suarez commented on routes/api-v1.0/user-route.js:
```diff
@@ -44,7 +49,7 @@ var UserRoute = (function(){
         //var deferred = Q.defer();
         newUser = request.body;
         //console.log(request.body);
```
Remove useless code.

david-suarez commented on routes/api-v1.0/user-route.js:
```diff
@@ -44,7 +49,7 @@ var UserRoute = (function(){
         newUser = request.body;
         //console.log(request.body);
-        if(newUser !== undefined) {
+        //if(newUser !== undefined) {
```
Remove commented code.

david-suarez commented on routes/api-v1.0/user-route.js:
```diff
@@ -52,9 +57,9 @@ var UserRoute = (function(){
-        }
+        //} else {
+        //    response.json('400', {'message': 'Bad Request'});
+        //}
```
Remove commented code

Adritay committed:
[feature] Create route for players
Description
========
Removed the comments from player-route file.
Added spaces at the end of the files.
